portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -99,7 +122,7 @@ private int getUserChoice(HttpServletRequest request) {
     }
 
     // Check that the input is between 1 and 20.
-    if (userChoice < 1 || userChoice > 20) {
+    if (userChoice < 1 || userChoice > 10) {
       return 5;
```

Comment:
Similarly 5 here should probably be a constant instead. e.g. `return INVALID_USER_SELECTION;`